### ISSUE TITLE
instability with 64 clients using cvmfs on the same node

### ISSUE THREAD

**Issue by dougbenjamin** [cvmfs/cvmfs#3674]
Hello,

When running 64 ATLAS pilot jobs all reading code from cvmfs at NERSC on the same node, (note the nodes have 2 CPU's 64 core/128 threads each), instability with CVMFS seen. The error reported for some of the jobs is "file not found"

When I scale back to 32 ATLAS pilot jobs on a NERSC compute node, the error goes away.

Regards,
Doug Benjamin

**Comment by vvolkl** [comment]
Hi Doug,

thanks for the report! Can you attach a bugreport tarball ( from `cvmfs_config bugreport`? ) It'll be important to see the syslog and know the cache configuration.

**Comment by dougbenjamin** [comment]
Is cvmfs_config a privileged executable? I don’t has such access to the NERSC HPC nodes
Sent from my iPhone

On Oct 10, 2024, at 1:58 PM, Valentin Volkl ***@***.***> wrote:

﻿

Hi Doug,

thanks for the report! Can you attach a bugreport tarball ( from cvmfs_config bugreport? ) It'll be important to see the syslog and know the cache configuration.

**Comment by vvolkl** [comment]
Right, this does need sudo privileges. Can you  access the syslog?  If yes, please send that, otherwise we should try to contact an admin.

**Comment by dougbenjamin** [comment]
I don’t have access. I will have create a test job that causes the failure and the open a ticket with NERSC support. Exactly what do I need to put in that ticket to get what you need?
Sent from my iPhone

On Oct 10, 2024, at 4:18 PM, Valentin Volkl ***@***.***> wrote:

﻿

Right, this does need sudo privileges. Can you access the syslog? If yes, please send that, otherwise we should try to contact an admin.

**Comment by jblomer** [comment]
Perhaps we can already gather some information from the `logbuffer` extended attribute. That would not require privileges.

**Comment by vvolkl** [comment]
That's a good point! You should be able to run 

```
attr -g logbuffer /cvmfs/atlas.cern.ch # replace with failing repository
```

in your job without privileges to get  a snippet of the syslog. If that's not practical, we would ideally want the `cvmfs_config bugreport` tarball from the affected node, or at least  the cvmfs configuration and the syslog.

**Comment by HereThereBeDragons** [comment]
@dougbenjamin any update on this? 
in case you cant run `sudo` the following commands would be useful:
- `cvmfs_config showconfig <repo>`
- something to show the size of the partition and its usage where the cvmfs cache is located ( e.g. `df -h` )

**Comment by dougbenjamin** [comment]
I am on vacation and will look into this when I return
Sent from my iPhone

On Nov 5, 2024, at 1:50 AM, HereThereBeDragons ***@***.***> wrote:

﻿

@dougbenjamin<https://github.com/dougbenjamin> any update on this?
in case you cant run sudo the following commands would be useful:

  *   cvmfs_config showconfig <repo>
  *   something to show the size of the partition and its usage where the cvmfs cache is located ( e.g. df -h )
